extensions/fairifier-rdf-extension/scripts/rdf-schema-vocabulary-manager.js

```diff
@@ -99,7 +99,11 @@ RdfPrefixesManager.prototype._hasPrefix = function(name){
 * Some utility functions
 */
 RdfPrefixesManager.isPrefixedQname = function(qname){
-	return qname.match(/[_a-zA-Z][-_a-zA-Z0-9]*:($|([_a-zA-Z][-_a-zA-Z0-9]*))$/);
+        if ((qname.split(':').length -1) < 3){
+            return qname.match(/[_a-zA-Z][-_a-zA-Z0-9]*:($|([_a-zA-Z][-_a-zA-Z0-9]*))$/);
+        } else{
+            return (qname.split(':')[0]+ ':' +  qname.split(':')[1]).match(/[_a-zA-Z][-_a-zA-Z0-9]*:($|([_a-zA-Z][-_a-zA-Z0-9]*))$/);
```

Comment:
The regex is repeated twice, maybe you can capture it in a variable to make it a bit more readable?

Reply:
sure will do